locale/pl/LC_MESSAGES/client.po

```diff
@@ -95,7 +95,7 @@ msgstr "Przykro nam o tym słyszeć. Jeśli nie ufasz już temu urządzeniu, to
 
 #: .es5/scripts/views/settings/clients.js:52
 msgid "Last active %(translatedTimeAgo)s"
-msgstr ""
+msgstr "Ostatnia aktywność: %(translatedTimeAgo)s"
```

Comment:
@splewako - I'll re-iterate what @vladikoff already said, I am sorry that we did not consult the l10n team before we used this approach. We assumed that since moment is a well-maintained and stable project that we could use their translations.

As @vladikoff mentioned, we have not been ignoring the bug, we have been actively trying to find a solution. Unfortunately, we were even less l10n aware 3 years ago when we wrote FxA's l10n infrastructure than we are now. At that time we made some bad decisions and our infrastructure does not at all handle 0/singular/plural forms. Investing the time to fix the entire stack (from extraction through to display) does not make sense for a single string. Instead, we reached for off-the-shelf translations for just the "time ago" portion of the string.

We perhaps ill-informed assumed that if we presented the string in the form of `Header: <translated time ago>` that would be able to be translated in all locales, the localizers are free to place both the `translated time ago` and header portion where it should be.